binder/postBuild

```diff
@@ -4,7 +4,7 @@ set -e
 # Install from sources
 # NB: If you want to use Jupytext on your binder, don't install it from source,
 # just add "jupytext" to your "binder/requirements.txt" instead.
-BUILD_JUPYTERLAB_EXTENSION=1 pip install .[myst]
+HATCH_BUILD_HOOKS_ENABLE=true pip install .
```

Comment:
This seems to be the default value when running `python -m build`. Do we need to specify it explicitly for dev installs?

Reply:
Good spot. That was vestigial, will update it